### ISSUE TITLE
IT Unstable case `TestGetIndexMVCC`

### ISSUE THREAD

**Issue by aytrack** [pingcap/tidb#30286]
## Bug Report

Please answer these questions before submitting your issue. Thanks!
in ci https://ci.pingcap.net/blue/organizations/jenkins/tidb_ghpr_coverage/detail/tidb_ghpr_coverage/1412/pipeline
```bash
2021-11-30T11:48:42.472Z] --- FAIL: TestGetIndexMVCC (1.37s)
[2021-11-30T11:48:42.472Z]     dbtestkit.go:72: 
[2021-11-30T11:48:42.472Z]         	Error Trace:	dbtestkit.go:72
[2021-11-30T11:48:42.472Z]         	            				http_handler_test.go:505
[2021-11-30T11:48:42.472Z]         	            				http_handler_test.go:748
[2021-11-30T11:48:42.472Z]         	Error:      	Received unexpected error:
[2021-11-30T11:48:42.472Z]         	            	commands out of sync. Did you run multiple statements at once?
[2021-11-30T11:48:42.472Z]         	Test:       	TestGetIndexMVCC
[2021-11-30T11:48:42.472Z]         	Messages:   	sql:create database tidb;, args:[]
[2021-11-30T11:48:42.472Z]     http_handler_test.go:501: 
[2021-11-30T11:48:42.472Z]         	Error Trace:	http_handler_test.go:501
[2021-11-30T11:48:42.472Z]         	            				panic.go:613
[2021-11-30T11:48:42.472Z]         	            				testing.go:741
[2021-11-30T11:48:42.472Z]         	            				dbtestkit.go:72
[2021-11-30T11:48:42.472Z]         	            				http_handler_test.go:505
[2021-11-30T11:48:42.472Z]         	            				http_handler_test.go:748
[2021-11-30T11:48:42.472Z]         	Error:      	Received unexpected error:
[2021-11-30T11:48:42.472Z]         	            	bad connection
[2021-11-30T11:48:42.472Z]         	Test:       	TestGetIndexMVCC
```

**Comment by mjonss** [comment]
Related to https://github.com/pingcap/tidb/issues/30249 ?

**Comment by djshow832** [comment]
> Related to #30249 ?

I think so. There are at least 5 relevant issues.

**Comment by hawkingrei** [comment]
Closed #30306. We can re-open it if the error happens again.
